src/admin.ts

```diff
@@ -35,28 +36,74 @@ interface ExecutionResult {
   };
 }
 
+interface Tenant {
+  authorized_party: string;
+  organization: string;
+  environment: string;
+  tenant: string;
+}
+
+interface TenantAPIParams {
+  envToken: string;
+}
+
+interface TenantAPIRequestParams extends TenantAPIParams {
+  environment: string;
+}
+
+interface TenantAPITenantParams extends TenantAPIRequestParams {
+  tenant: string;
+}
+interface TenantAPITenantChangeParams extends TenantAPITenantParams {
+  description?: string;
+}
+
+interface SessionAPIParams {
+  envToken: string;
+  environment: string;
+  tenant: string;
+}
+
 const ADMIN_URI =
   "http://se2-controlplane-service.suborbital.svc.cluster.local:8081";
 
+const API_URI = "https://api.stg.suborbital.network";
```

Comment:
I removed the custom bits and pointed the fallback address to staging.

If we wanted to keep backwards compat we could probably keep the controlplane URI as a default (i'm setting it manually from the demo anyway), but it may not matter given the changes abound.